src/styles/todo-list.css

```diff
@@ -114,3 +114,7 @@
     background: none;
   }
 }
+
+input {
```

Comment:
Using the 'input' element selector with a global scope could potentially apply this style to all 'input' elements in the project, including those outside of the todo list. It's better to use a class or id selector to specifically target the desired inputs.